homework/jkim237/forms/tables/forms.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```

Comment:
There is a template for this homework!

Please move this code to the path `fe-soon/homework/jkim237/20201012/2-forms.html`